.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
Any objections to adding dependendabot @dundargoc @jamessan @bfredl  ?

Reply:
It's very noisy, and opens up a new can of worms (frequent security advisories, Github changing how dependabot works).

I would prefer not to add this just for a single package. (In fact, I would prefer this whole thing to live outside our repo.)

Reply:
Dependabot should help with security advisories, not increase them. Assuming we want to use the GHA packages that we are using.

Reply:
If we use it for _all_ actions, it may indeed help with catching outdated actions earlier. I'd prefer to discuss (and tune) this as a separate PR, though, not shoehorned into the winget update.

Reply:
Ok, @vedantmgoyal2009 would you mind sending this depedabot change as a separate pr?